### DISCUSSION TITLE
Enable/Disable User registration or Invite Function

### DISCUSSION THREAD

**Discussion by BelieveInBunny** [torrust/torrust-index-gui#91] [Feature requests]
Nice to have an option to disable Registration or only with invite code. Or Admins can add Users Accounts.

Related to: #9

**Comment by da2ce7** [comment]
relates to:
- #284

**Comment by josecelano** [comment]
Relates to: https://github.com/torrust/torrust-index/issues/430

Hi, @BelieveInBunny, you can disable registration by removing the `[registration]` section from the configuration.

Invitations or manually adding users by admins are features that have not yet been implemented.

**Comment by josecelano** [comment]
Hi @BelieveInBunny, disable registration was implemented, and the invitations feature implementation is defined [here](https://github.com/torrust/torrust-index-gui/discussions/285).
